substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfAbbrevSectionImpl.java

```diff
@@ -927,6 +927,10 @@ private int writeBuiltInUnitAbbrev(@SuppressWarnings("unused") DebugContext cont
         pos = writeFlag(DwarfDebugInfo.DW_CHILDREN_yes, buffer, pos);
         pos = writeAttrType(DwarfDebugInfo.DW_AT_language, buffer, pos);
         pos = writeAttrForm(DwarfDebugInfo.DW_FORM_data1, buffer, pos);
+        pos = writeAttrType(DwarfDebugInfo.DW_AT_comp_dir, buffer, pos);
```

Comment:
I think also need to emit the corresponding data in `com.oracle.objectfile.elf.dwarf.DwarfInfoSectionImpl#writeBuiltInUnit`

Reply:
Yes, @gilles-duboscq  is right. See my comment below.